src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -107,6 +107,11 @@ configurationRegistry.registerConfiguration({
 			'default': 'right',
 			'description': nls.localize({ comment: ['This is the description for a setting. Values surrounded by single quotes are not to be translated.'], key: 'editorOpenPositioning' }, "Controls where editors open. Select 'left' or 'right' to open editors to the left or right of the current active one. Select 'first' or 'last' to open editors independently from the currently active one.")
 		},
+		'workbench.editor.reuseIfOpen': {
```

Comment:
No worries. To update this PR you just push to your branch `ddotlic:master`. Thats it.